readthedocs/proxito/views/serve.py

```diff
@@ -1034,6 +1047,16 @@ def changefreqs_generator():
             content_type='application/xml',
         )
 
+    def _get_project(self):
+        # Method used by the CDNCacheTagsMixin class.
+        return self.request.unresolved_domain.project
+
+    def _get_version(self):
+        # Method used by the CDNCacheTagsMixin class.
+        # This view isn't explicitly mapped to a version,
+        # TODO: refactor how we set cache tags to avoid this.
+        return None
```

Comment:
I'm tracking how to improve this at https://github.com/readthedocs/readthedocs.org/issues/10124